src/components/Oauth/NoWorkspaceEntry/NoWorkspaceOauthFlow.tsx

```diff
@@ -49,7 +49,7 @@ export function NoWorkspaceOauthFlow({
       );
       setOAuthCallbackURL(url);
     } catch (err: any) {
-      console.error(err);
+      console.error('Could not fetch OAuthCallback URL', { err });
```

Comment:
I realized that this ` fetchOAuthCallbackURL` was actually a poorly named function, since it can be confused with the callback URL that that Salesforce redirects to. Can you rename it `fetchOAuthPopupURL` or something like that, and revise this error message to say 'Could not fetch OAuth popup URL'